### ISSUE TITLE
Web - Search Page - Missing Submit Button

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#87]
There is no submit button for the search box. 

Search - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/search

**Comment by KyleLivingstonDOL** [comment]
ZTI, please build in a "Search" button, under the search bar in the middle. Please make it the same size as the Previous/Next step button. Please make it blue.

You can use the translations that we provided for "search".

**Comment by Abychalil** [comment]
Search Button Added to submit the search key word, in ll three languages, English, french and Spanish

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for adding the language-specific (English, Spanish, and French) search button to the correct screen. 

Will look at it when the next version/build is sent, and I  have time to test.

**Comment by jfc3-dol** [comment]
There is now a "Search" button for the search page. The only issue I found is my screen reader JAWS now reads "Search submit button button", which is confusing.

I think that is because you've added the word "submit button" to the ARIA- LABEL and JAWS adds one because the element is a button.

Closing this issue and opening a new one to remove the extra "button" being read.
